src/components/overlays/tooltip/Tooltip.vue

```diff
@@ -38,19 +40,19 @@ const {
 
 <template>
   <div
-    ref="trigger"
+    ref="activator"
     :class="css.wrapper"
     :data-tooltip-disabled="disabled"
     @mouseover="onMouseOver()"
     @mouseleave="onMouseLeave()"
   >
-    <div :class="css.trigger">
-      <slot :open="open" />
+    <div :class="css.activator">
+      <slot name="activator" :open="open" />
```

Comment:
I was a bit divided over switching to vuetify's approach for this, but this will make migration easier for us for certain.